### PR TITLE
[RFR] #168 + updated BBApplication::_initAnnotationReader to register a custom loader for backbee's annotations

### PR DESCRIPTION
**Pull request by eric-chau**
cf #168 for more details

### CODE REVIEW

File: BBApplication.php [hunk 1]
```diff
@@ -1072,11 +1072,11 @@ private function _initAnnotationReader()
         AnnotationRegistry::registerAutoloadNamespaces(array(
             'Symfony\Component\Validator\Constraint' => $this->getVendorDir().'/symfony/symfony/src/',
             'JMS\Serializer\Annotation' => $this->getVendorDir().'/jms/serializer/src/',
-            'BackBee\Installer\Annotation' => $this->getBaseDir(),
-            'BackBee' => $this->getBaseDir(),
             //'Doctrine\ORM\Mapping' => $this->getVendorDir() . '/doctrine/orm/lib/'
         ));
 
+        AnnotationRegistry::registerLoader('class_exists');
```
Comment (mickaelandrieu) [inline BBApplication.php]: are you sure ? o_O

Comment (eric-chau) [inline BBApplication.php]: Yeah =)

**Comment by eric-chau** [comment]
error is due to hhvm, should we allow error for hhvm? Also, our tests exceed travis max time execution (~50 min) so 5.4, 5.5 and 5.6 test suite are not complete. I think this is due to travis had alot of load. Can we re-run it?

**Comment by mickaelandrieu** [comment]
Yes, week end.

**Comment by eric-chau** [comment]
hm, week end? lol Can you re-run tests please? Or tell me how to do it :P thanks

**Comment by mickaelandrieu** [comment]
private joke with "Yes, we can" sorry.

I'll relaunch travis (I believed you also can too).

**Comment by eric-chau** [comment]
Every tests are now all green.

**Pull request merged**